.github/workflows/linux.yml

```
@@ -369,6 +369,9 @@ jobs:
           - name: 'API tests'
             cmd: 'tests/python_tests/test_continuous_batching.py tests/python_tests/test_generation_config.py tests/python_tests/test_sampling.py tests/python_tests/test_text_streamer.py'
             timeout: 60
+          - name: 'Rag tests'
+            cmd: 'tests/python_tests/test_rag.py'
+            timeout: 15
```

```suggestion
            timeout: 30
```
Align with windows. Imagine someone not familiar with the tests. If they happened to read Windows first, they would expect it to be the same for the other two.

---

Aligned